### PR TITLE
Fix for mobile view

### PR DESCRIPTION
**Pull request by noemiperez**

### CODE REVIEW

**Comment by noemiperez** [comment]
Done

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "printdeal-styleguide",
   "description": "Printdeal Styleguide",
-  "version": "1.8.1",
+  "version": "1.9.0",
```
Comment (bartveneman) [inline package.json]: Should be 1.8.2, since you are doing a fix, not adding a new feature. See http://semver.org/

Comment (noemiperez) [reply package.json]: True, changing

**Comment by noemiperez** [comment]
Done!

**Pull request merged**